### ISSUE TITLE
[Bug]: 1.19.3 Support. 

### ISSUE THREAD

**Issue by Lebyy** [EncryptSL/LiteEco#14]
### What happened?

Hello there! I encountered this error after updating my server to 1.19.3 yesterday. 

### Plugin Version

1.1.7-SNAPSHOT

### You detected problem on this server platform ?

PurpurMC

### Relevant log output

```shell
[05:06:58] [Server thread/INFO]: [LiteEco] Registering commands with Cloud Command Framework !
[05:06:58] [Server thread/ERROR]: Error occurred while enabling LiteEco v1.1.7-SNAPSHOT (Is it up to date?)
encryptsl.cekuj.net.cloud.bukkit.BukkitCommandManager$BrigadierFailureException: Could not initialize Brigadier mappings. Reason: paper brigadier initialization failure (null)
	at encryptsl.cekuj.net.cloud.paper.PaperCommandManager.registerBrigadier(PaperCommandManager.java:134) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.LiteEco.registerCommands(LiteEco.kt:89) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.LiteEco.onEnable(LiteEco.kt:68) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[purpur-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:372) ~[purpur-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:550) ~[purpur-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R2.CraftServer.enablePlugin(CraftServer.java:624) ~[purpur-1.19.3.jar:git-Purpur-1861]
	at org.bukkit.craftbukkit.v1_19_R2.CraftServer.enablePlugins(CraftServer.java:538) ~[purpur-1.19.3.jar:git-Purpur-1861]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[purpur-1.19.3.jar:git-Purpur-1861]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1115) ~[purpur-1.19.3.jar:git-Purpur-1861]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[purpur-1.19.3.jar:git-Purpur-1861]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.ExceptionInInitializerError
	at encryptsl.cekuj.net.cloud.bukkit.internal.MinecraftArgumentTypes$ArgumentTypeGetterImpl.<init>(MinecraftArgumentTypes.java:109) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.bukkit.internal.MinecraftArgumentTypes$ArgumentTypeGetterImpl.<init>(MinecraftArgumentTypes.java:103) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.bukkit.internal.MinecraftArgumentTypes.<clinit>(MinecraftArgumentTypes.java:78) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.bukkit.BukkitBrigadierMapper.registerMappings(BukkitBrigadierMapper.java:82) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.bukkit.BukkitBrigadierMapper.<init>(BukkitBrigadierMapper.java:75) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.paper.PaperBrigadierListener.<init>(PaperBrigadierListener.java:72) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.paper.PaperCommandManager.registerBrigadier(PaperCommandManager.java:128) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	... 11 more
Caused by: java.lang.IllegalStateException: Could not find Registry Registry field
	at encryptsl.cekuj.net.cloud.bukkit.internal.RegistryReflection.lambda$static$2(RegistryReflection.java:71) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at java.util.Optional.orElseThrow(Optional.java:403) ~[?:?]
	at encryptsl.cekuj.net.cloud.bukkit.internal.RegistryReflection.<clinit>(RegistryReflection.java:71) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.bukkit.internal.MinecraftArgumentTypes$ArgumentTypeGetterImpl.<init>(MinecraftArgumentTypes.java:109) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.bukkit.internal.MinecraftArgumentTypes$ArgumentTypeGetterImpl.<init>(MinecraftArgumentTypes.java:103) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.bukkit.internal.MinecraftArgumentTypes.<clinit>(MinecraftArgumentTypes.java:78) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.bukkit.BukkitBrigadierMapper.registerMappings(BukkitBrigadierMapper.java:82) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.bukkit.BukkitBrigadierMapper.<init>(BukkitBrigadierMapper.java:75) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.paper.PaperBrigadierListener.<init>(PaperBrigadierListener.java:72) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	at encryptsl.cekuj.net.cloud.paper.PaperCommandManager.registerBrigadier(PaperCommandManager.java:128) ~[LiteEco-1.1.7-SNAPSHOT.jar:?]
	... 11 more
[05:06:58] [Server thread/INFO]: [LiteEco] Disabling LiteEco v1.1.7-SNAPSHOT
[05:06:58] [Server thread/INFO]: [LiteEco] Plugin is disabled
```

**Comment by EncryptSL** [comment]
Fixed by commit: https://github.com/EncryptSL/LiteEco/commit/26b5f57ffd1347f1fe0729c7c7366f934f7317ee

New version released there: https://www.spigotmc.org/resources/liteeco-simple-economy-plugin-1-18-x-1-19.101934/update?update=481899

**Comment by Lebyy** [comment]
> Fixed by commit: [26b5f57](https://github.com/EncryptSL/LiteEco/commit/26b5f57ffd1347f1fe0729c7c7366f934f7317ee)
> 
> New version released there: https://www.spigotmc.org/resources/liteeco-simple-economy-plugin-1-18-x-1-19.101934/update?update=481899

Thank you so much for the blazingly fast response and update!
